### PR TITLE
Vendor gardener@v1.53.0

### PR DESCRIPTION
**Pull request by ary1992**
**How to categorize this PR?**

/area open-source
/kind enhancement
/platform vsphere

**What this PR does / why we need it**:
Vendor `gardener@v1.53.0`
https://github.com/gardener/gardener/releases/tag/v1.53.0
https://github.com/gardener/gardener/releases/tag/v1.52.0
https://github.com/gardener/gardener/releases/tag/v1.51.0
https://github.com/gardener/gardener/releases/tag/v1.50.0
https://github.com/gardener/gardener/releases/tag/v1.49.0

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
The following dependency is updated:
- github.com/gardener/gardener: v1.48.0 -> v1.53.0
- k8s.io/* : v0.23.2 -> v0.24.3
```

### CODE REVIEW

File: charts/gardener-extension-validator-vsphere/charts/runtime/templates/deployment.yaml [hunk 1]
```diff
@@ -46,6 +46,9 @@ spec:
         {{- if .Values.global.kubeconfig }}
         - --kubeconfig=/etc/gardener-extension-validator-vsphere/kubeconfig/kubeconfig
         {{- end }}
+        {{- if .Values.metricsPort }}
```
Comment (briantopping) [inline charts/gardener-extension-validator-vsphere/charts/runtime/templates/deployment.yaml:49]: I'm guessing you meant to say `{{- if .Values.global.metricsPort }}` here.

**Review by briantopping — COMMENTED** [review]
One comment on a Helm file, but otherwise looks great to me. This is a very substantial set of upgrades, thanks for your efforts on them!

File: charts/gardener-extension-validator-vsphere/charts/runtime/templates/deployment.yaml (hunk 1 above)
Comment (ary1992) [reply charts/gardener-extension-validator-vsphere/charts/runtime/templates/deployment.yaml]: Thank you!

**Review by briantopping — APPROVED** [review]
/lgtm thanks @ary1992!

**Pull request merged**